pkg/cmd/deploy/requests.go

```diff
@@ -112,10 +125,11 @@ func (cmd *DeployCmd) doOrigin(client *apiapp.Client, clientOrigin *apiori.Clien
 	if conf.Template == "javascript" || conf.Template == "typescript" {
 		return nil
 	}
+
 	var addresses []string
 	var DefaultOrigin = [1]string{"httpbin.org"}
 
-	if conf.Mode == "compute" {
+	if strings.ToLower(conf.Mode) == "compute" {
```

Comment:
Technically we do not need this, because when we save them we already should be setting them to lowercase 🤔

Reply:
I believe it is a valid validation, do not trust the user (it can change and will 😂)

Reply:
@jotanarciso makes sense 😆